### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by thomasridd**
Dependency update to re-start the project

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ parameters:
 
   node-version:
     type: string
-    default: 18.15-browsers
+    default: 18.17-browsers
```
Comment (SimonMitchellMOJ) [inline .circleci/config.yml:21]: can go to `18.18-browsers`

File: Dockerfile [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # Stage: base image
-FROM node:18.16-bullseye-slim as base
+FROM node:20.8-bullseye-slim as base
```
Comment (SimonMitchellMOJ) [inline Dockerfile:2]: we need to stay on the LTS version so need to be `FROM node:18.18-bullseye-slim as base`

File: .circleci/config.yml (hunk 1 above)
Comment (thomasridd) [reply .circleci/config.yml]: done

File: Dockerfile (hunk 2 above)
Comment (thomasridd) [reply Dockerfile]: done - though this comes from a PR kicked off by renovate-bot https://github.com/ministryofjustice/hmpps-authorization/pull/8

**Review by SimonMitchellMOJ — APPROVED** [review]
LGTM

**Pull request merged**